tests/testthat/test-select_nse.R

```diff
@@ -138,3 +138,22 @@ test_that(".select_nse: works with function and namespace", {
   out <- fun(insight::find_predictors(model, effects = "fixed", flatten = TRUE))
   expect_identical(out, iris["Petal.Width"])
 })
+
+test_that(".select_nse: allow character vector with :", {
```

Comment:
Can you add a test for a case where one does `col1:col2` but `col1` comes after `col2` in the dataset?